test/parallel_api/experimental/asynch.pass.cpp

```diff
@@ -114,61 +115,69 @@ void test2_with_buffers()
         EXPECT_TRUE(fabs(result1-expected1) <= 0.001f && fabs(result2-expected1) <= 0.001f, "wrong effect from async test (II) with sycl buffer");
 }
 
-void test_with_usm()
+// TODO: Extend tests by checking true async behavior in more detail
+template <sycl::usm::alloc alloc_type>
+void
+test_with_usm()
 {
     cl::sycl::queue q;
-    const int n = 1024;
-    const int n_small = 13;
 
-    // ASYNC TEST USING USM //
-    // TODO: Extend tests by checking true async behavior in more detail
-    {
-        // Allocate space for data using USM.
-        uint64_t* data1 =
-            static_cast<uint64_t*>(cl::sycl::malloc_shared(n * sizeof(uint64_t), q.get_device(), q.get_context()));
-        uint64_t* data2 =
-            static_cast<uint64_t*>(cl::sycl::malloc_shared(n * sizeof(uint64_t), q.get_device(), q.get_context()));
-
-        // Initialize data
-        for (int i = 0; i != n - 1; ++i)
+    constexpr int n = 1024;
+    constexpr int n_small = 13;
+
+    // Initialize data
+    auto prepare_data = [](int n, uint64_t* data1, uint64_t* data2)
```

Comment:
Generally, it's a good idea to put `std` namespace for the types from C++ standard library (in our case `::std::`) even if they were missed previously.

Reply:
Fixed for: uint8_t, uint16_t, uint32_t, uint64_t, int32_t, int64_t in tests